pkg/rpcclient/actor/waiter.go

```diff
@@ -222,23 +223,45 @@ func (w *EventWaiter) WaitAny(ctx context.Context, vub uint32, hashes ...util.Ui
 		if wsWaitErr != nil {
 			res, waitErr = w.polling.WaitAny(ctx, vub, hashes...)
 			if waitErr != nil {
-				waitErr = fmt.Errorf("WS waiter error: %w; simple waiter error: %v", wsWaitErr, waitErr)
+				// Wrap the poll-based error, it's more important.
+				waitErr = fmt.Errorf("event-based error: %v; poll-based waiter error: %w", wsWaitErr, waitErr)
 			}
 		}
 	}()
 	bRcvr := make(chan *block.Block)
 	aerRcvr := make(chan *state.AppExecResult)
+	wg := sync.WaitGroup{}
```

Comment:
I'd rather use an `exit chan error` here, make unsub threads push their errors to it...

Reply:
Fixed.